<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Lobby</class>
 <widget class="QDialog" name="Lobby">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>581</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>581</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>581</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(0, 0, 0, 255), stop:0.142045 rgba(103, 0, 0, 255), stop:0.284091 rgba(165, 0, 0, 255), stop:0.426136 rgba(255, 0, 0, 255), stop:0.573864 rgba(255, 59, 0, 255), stop:0.710227 rgba(255, 103, 1, 255), stop:0.835227 rgba(255, 162, 0, 255), stop:0.935 rgba(243, 207, 0, 255));</string>
  </property>
  <widget class="QLabel" name="titolo">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>63</y>
     <width>500</width>
     <height>97</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>500</width>
     <height>97</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>500</width>
     <height>97</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Swis721 BlkOul BT</family>
     <pointsize>65</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
     <underline>false</underline>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="text">
    <string>UpWords</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QStackedWidget" name="swPage">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>180</y>
     <width>500</width>
     <height>350</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>500</width>
     <height>350</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>500</width>
     <height>350</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;</string>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QPushButton" name="btGiocatoreSingolo">
     <property name="geometry">
      <rect>
       <x>147</x>
       <y>48</y>
       <width>206</width>
       <height>37</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>206</width>
       <height>37</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>206</width>
       <height>37</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(155, 0, 0, 255), stop:1 rgba(24, 0, 0, 255));
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Giocatore Singolo</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btTreGiocatori">
     <property name="geometry">
      <rect>
       <x>147</x>
       <y>182</y>
       <width>206</width>
       <height>37</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>206</width>
       <height>37</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>206</width>
       <height>37</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(254, 142, 28, 255), stop:0.55 rgba(244, 126, 6, 255), stop:1 rgba(172, 0, 0, 255));</string>
     </property>
     <property name="text">
      <string>Tre Giocatori</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btDueGiocatori">
     <property name="geometry">
      <rect>
       <x>147</x>
       <y>115</y>
       <width>206</width>
       <height>37</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>206</width>
       <height>37</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>206</width>
       <height>37</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(254, 142, 28, 255), stop:0.55 rgba(244, 126, 6, 255), stop:1 rgba(172, 0, 0, 255));</string>
     </property>
     <property name="text">
      <string>Due Giocatori</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btQuattroGiocatori">
     <property name="geometry">
      <rect>
       <x>147</x>
       <y>249</y>
       <width>206</width>
       <height>37</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>206</width>
       <height>37</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>206</width>
       <height>37</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(254, 142, 28, 255), stop:0.55 rgba(244, 126, 6, 255), stop:1 rgba(172, 0, 0, 255));</string>
     </property>
     <property name="text">
      <string>Quattro Giocatori</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QLineEdit" name="lineEdit_Giocatore4">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>218</y>
       <width>280</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>280</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>280</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="statusTip">
      <string notr="true"/>
     </property>
     <property name="whatsThis">
      <string/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_Giocatore3">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>153</y>
       <width>130</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>130</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>130</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Giocatore 3:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_Giocatore4">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>218</y>
       <width>130</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>130</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>130</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Giocatore 4:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Giocatore3">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>153</y>
       <width>280</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>280</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>280</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_Giocatore2">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>88</y>
       <width>130</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>130</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>130</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Giocatore 2:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_Giocatore1">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>23</y>
       <width>130</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>130</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>130</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Giocatore 1:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btStart">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>290</y>
       <width>140</width>
       <height>41</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>140</width>
       <height>41</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>140</width>
       <height>41</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(254, 142, 28, 255), stop:0.55 rgba(244, 126, 6, 255), stop:1 rgba(172, 0, 0, 255));</string>
     </property>
     <property name="text">
      <string>START</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Giocatore1">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>23</y>
       <width>280</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>280</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>280</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Giocatore2">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>88</y>
       <width>280</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>280</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>280</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="btIndietro">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>290</y>
       <width>140</width>
       <height>41</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>140</width>
       <height>41</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>140</width>
       <height>41</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(254, 142, 28, 255), stop:0.55 rgba(244, 126, 6, 255), stop:1 rgba(172, 0, 0, 255));</string>
     </property>
     <property name="text">
      <string>INDIETRO</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
